src/Appwrite/Auth/OAuth2/Google.php

```diff
@@ -53,7 +53,9 @@ public function getLoginURL(): string
             'redirect_uri' => $this->callback,
             'scope' => \implode(' ', $this->getScopes()),
             'state' => \json_encode($this->state),
-            'response_type' => 'code'
+            'response_type' => 'code',
+            'access_type' => 'offline',
+            'prompt' => 'consent'
```

Comment:
Do we have to add this `'prompt' => 'consent'`? If not, let's leave it out to avoid changing the behavior.
```suggestion
```